iidm/iidm-serde/src/main/resources/xsd/iidm_equipment_V1_15.xsd

```diff
@@ -208,10 +208,14 @@
     </xs:complexType>
     <xs:complexType name="ReactiveCapabilityCurve">
         <xs:sequence>
+            <xs:element name="property" type="iidm:Property" minOccurs="0" maxOccurs="unbounded" />
             <xs:element name="point" minOccurs="2" maxOccurs="unbounded" type="iidm:Point"/>
```

Comment:
The properties support on `Point`s is handled in the SerDe, but not in the XSDs.